src/build/mod.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
+    external_cli::{cargo, rustup, wasm_bindgen},
+    manifest::package_name,
+};
+
+pub use self::args::BuildArgs;
+
+mod args;
+
+pub fn build(args: &BuildArgs) -> anyhow::Result<()> {
+    if args.is_web() {
+        ensure_web_setup()?;
+    }
+
+    let cargo_args = args.cargo_args_builder();
+
+    if args.is_web() {
+        println!("Building for WASM...");
+        cargo::build::command().args(cargo_args).status()?;
```

Comment:
We should probably handle this status, in case the exit code is non-zero. (We're currently handling if Cargo cannot be spawned, but not if it exits with an error.) I like to use `anyhow`'s `ensure!()` macro:

```rust
let status = cargo::build::command().args(cargo_args).status()?;
ensure!(status.success());
```

Reply:
Good point, since this is a common thing we need to do, I added a helper trait with `.ensure_status()` which simplifies this